### ISSUE TITLE
Make slider accessible

### ISSUE THREAD

**Issue by EldRoos** [sebgroup/react-components#79]
**Is your feature request related to a problem? Please describe.**
Missing hover and focus state for slider component. Makes it hard if not impossible to follow the new accessibility law. People with disabilities have hard time to step through components when they do not look different when selected.

**Describe the solution you'd like**
* Have highlights around thumb when it is selected (focused with tab/finger/mouse). Use box-shadow 3px lightblue like many other components (0.15s ease-in-out if fit). Probably use "focus" class for this.
* Make the thumb increase in size when hovering it (with a short animation). Extra size should be 5%. To compensate with the increased size make the non hovered state 5% smaller. See avanza.se for example (scroll down a bit to the green sliders).

**Describe alternatives you've considered**
Use number 1 of these:
![SliderHighlights](https://user-images.githubusercontent.com/14058437/67285304-99fab080-f4d7-11e9-8cba-90dfda552a80.png)
![SliderHover](https://user-images.githubusercontent.com/14058437/67285437-daf2c500-f4d7-11e9-9dd9-bcca32713306.png)

**Additional context**
Ulrika Dettner-Larsson and Henrik Hoglund have approved this change.

**Comment by EldRoos** [comment]
Finally got react-components to build. Looks like disabled slider have hover effect, it should not.
